src/irmin-pack/checks.ml

```diff
@@ -0,0 +1,225 @@
+include Checks_intf
+module T = Irmin.Type
+module IO = IO.Unix
+open Lwt.Infix
+
+let ( // ) = Filename.concat
+
+let current_version = `V2
+
+(** Only works for layered stores that use the default names for layers. *)
```

Comment:
Seems like we should put these somewhere in `irmin-layers`, but this will do for now.

Reply:
Are you referring to the default names of the layers? For now they are in [`irmin_pack_layers`](https://github.com/mirage/irmin/blob/master/src/irmin-pack/irmin_pack_layers.ml#L41) indeed, we will probably move a part of the configuration to `irmin-layers` once we start writing layers store for other backends.

Reply:
Indeed :+1: